README.md

```diff
@@ -92,158 +76,166 @@ Additionally, the device and any additional complementary hardware components sh
     }
     ```
 
-3. Configure the API key for your Pelion Portal account.
+1. Configure the API key for your Pelion Portal account.
 
-     If you don't have an API key available, then login in [Pelion IoT Platform portal](https://portal.mbedcloud.com/), navigate to 'Access Management', 'API keys' and create a new one. Then specify the API key as global `mbed` configuration:
+   If you don't have an API key available, then log in to the [Pelion IoT Platform portal](https://portal.mbedcloud.com/), navigate to `Access Management` and `API keys`, and create a new one. Then specify the API key as the global `mbed` configuration:
 
     ```
     $ mbed config -G CLOUD_SDK_API_KEY <your-api-key>
     ```
 
-4. Install the device management certificate:
+1. Install the device management certificate:
 
     ```
     $ mbed dm init -d "<your company name.com>" --model-name "<product model identifier>"
     ```
 
-This will create your private/public key pair and also initialize various .c files with these credentials, so you can use Connect and (firmware) Update device management features.
-
+This creates your private and public key pair and also initialize various `.c` files with these credentials, so you can use Connect and (firmware) Update device management features.
```

Comment:
Should we link to docs about Connect and Update?

Note to self: Change "initialize" to "initializes".